File: wireless/v1/command/create-binary-example-1/create-binary-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

command = @client.wireless.commands.create(
                                      command_mode: 'binary',
                                      sim: 'AliceSmithSmartMeter',
                                      command: 'SGVsbG8sIE1hY2hpbmUh=='
                                    )

puts command.sid

File: wireless/v1/command/create-binary-example-1/create-binary-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$command = $twilio->wireless->v1->commands
                                ->create("SGVsbG8sIE1hY2hpbmUh==", // command
                                         array(
                                             "commandMode" => "binary",
                                             "sim" => "AliceSmithSmartMeter"
                                         )
                                );

print($command->sid);